build/Settings.cs

```diff
@@ -25,6 +25,7 @@ private static string config(string @default = null, [CallerMemberName] string k
         public const string DotnetProjectTemplatesVersion = "4.0.1964";
         public const string TemplateJsonVersion = "3.1.1648";
 
+        public static readonly string SBOMManifestToolPath = Path.GetFullPath("../ManifestTool/Microsoft.ManifestTool.dll");
```

Comment:
nit: Please add an empty line between `SBOMManifestToolPath` and `SrcProjectPath`.